tests/installation/configure_bls.pm

```diff
@@ -36,6 +36,8 @@ sub run {
     send_key 'spc', wait_screen_change => 1;
     send_key_until_needlematch 'inst-bootloader-systemd-boot-selected', 'down' if is_bootloader_sdboot;
     send_key_until_needlematch 'inst-bootloader-grub2-bls-selected', 'down' if is_bootloader_grub2_bls;
+    send_key_until_needlematch 'inst-bootloader-grub2-efi-selected', 'up' if is_bootloader_grub2 && check_var('UEFI', 1);
+    send_key_until_needlematch 'inst-bootloader-grub2-selected', 'up' if is_bootloader_grub2 && check_var('UEFI', 0);
```

Comment:
```suggestion
    send_key_until_needlematch get_var('UEFI') ? 'inst-bootloader-grub2-efi-selected' : 'inst-bootloader-grub2-selected', 'up' if is_bootloader_grub2;
```

But it would be nicer to just put the tag 'inst-bootloader-grub2-selected' on any 'inst-bootloader-grub2-efi-selected' needle additionally and just match on that